autoreject/autoreject.py

```diff
@@ -663,11 +663,12 @@ def fit(self, epochs):
                 threshes.update(sub_ar.threshes_)
                 bad_segments += sub_ar.bad_segments
                 fix_log += sub_ar.fix_log
-            # assemble stuff and return
-            sub_ar.threshes_ = threshes
-            sub_ar._local_reject.fix_log = fix_log
-            sub_ar._drop_log = bad_segments
-            return sub_ar
+            # assemble stuff, update and return self
+            self.threshes_ = threshes
+            # XXX @jasmainak this needs to be retought
+            self._local_reject.fix_log = fix_log
```

Comment:
@jasmainak we need to set the attribute directly-- not via ._local_reject-- otherwise we'll end up in trouble. Ok?